inc/Editor/BlockManagement/ConfigRegistry.php

```diff
@@ -96,7 +96,7 @@ public static function register_block( array $user_config = [] ): bool|WP_Error
 
 			foreach ( array_keys( $to_query->get_input_schema() ) as $to ) {
 				if ( ! isset( $from_output_schema['type'][ $to ] ) ) {
-					return self::create_error( $block_title, sprintf( 'Cannot map key "%s" from %s query', esc_html( $to ), $from_query_type ) );
+					return self::create_error( $block_title, sprintf( 'Cannot map key "%1$s" from %2$s query. The display query requires a "%1$s" key as an input, but it is not present in the output schema for the %2$s query. Try adding "%1$s" to the output schema for the %2$s query.', esc_html( $to ), $from_query_type, $from_query_type, esc_html( $to ) ) );
```

Comment:
Do you need those last added args to `sprintf`?

Reply:
Good catch! Removed those.